addons/io_scene_gltf2/io/exp/gltf2_io_image_data.py

```diff
@@ -19,6 +19,13 @@ class ImageData:
     # FUTURE_WORK: as a method to allow the node graph to be better supported, we could model some of
     # the node graph elements with numpy functions
 
+    extension_for_mime = {
```

Comment:
We end up with multiple spots with very similar mappings... should be centralized somehow